pkg/common/utils/k8s/client.go

```diff
@@ -229,6 +229,20 @@ func GetConfigMap(ctx context.Context, k8scient client.Client, namespace, name s
 	return &configMap, nil
 }
 
+// CheckSecretExist check if the secret is in the namespace.
+func CheckSecretExist(ctx context.Context, k8scient client.Client, namespace string, secrets []dorisv1.Secret) {
```

Comment:
This func is not necessary. After checking, it only prints the log and has no effect on subsequent operations. Checking whether it exists can be done later.

Reply:
I moved `CheckSecretExist` to after `k8s.ApplyStatefulSet`. I think this check is necessary. If the secret is not created, there will be no error log below in the operator log, and the pod will always be in the ContainerCreating state.
<img width="1449" alt="image" src="https://github.com/user-attachments/assets/7998a555-aa98-4f4c-82ff-af20267ecb54">